### PR TITLE
DOMA-1976 added errors to `doc` section of `ConfirmPhoneActionService`

### PR DESCRIPTION
**Pull request by AntonAL**

### CODE REVIEW

File: apps/condo/domains/user/schema/ConfirmPhoneActionService.js [hunk 1]
```diff
@@ -142,6 +150,13 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
         {
             access: true,
             schema: 'startConfirmPhoneAction(data: StartConfirmPhoneActionInput!): StartConfirmPhoneActionOutput',
+            doc: {
+                summary: 'Creates ConfirmPhoneAction and sends confirmation message',
```
Comment (pahaz) [inline apps/condo/domains/user/schema/ConfirmPhoneActionService.js:154]:
```suggestion
                summary: 'Send confirmation phone SMS message and return confirmation token. You can use the token for completeConfirmPhoneAction mutation. And then use the token in other mutations to prove that the phone number is verified',
```

File: apps/condo/domains/user/schema/ConfirmPhoneActionService.js [hunk 2]
```diff
@@ -191,6 +206,13 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
         {
             access: true,
             schema: 'resendConfirmPhoneActionSms(data: ResendConfirmPhoneActionSmsInput!): ResendConfirmPhoneActionSmsOutput',
+            doc: {
+                summary: 'Tries to send again a message to confirm phone',
```
Comment (pahaz) [inline apps/condo/domains/user/schema/ConfirmPhoneActionService.js:210]:
```suggestion
                summary: 'Resend the confirm phone SMS message for existing token',
```

File: apps/condo/domains/user/schema/ConfirmPhoneActionService.js [hunk 3]
```diff
@@ -233,6 +255,16 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
         {
             access: true,
             schema: 'completeConfirmPhoneAction(data: CompleteConfirmPhoneActionInput!): CompleteConfirmPhoneActionOutput',
+            doc: {
+                summary: 'Final step in phone number confirmation',
```
Comment (pahaz) [inline apps/condo/domains/user/schema/ConfirmPhoneActionService.js:259]:
```suggestion
                summary: 'The final step of a phone number confirmation. You should use the token from startConfirmPhoneAction and a secret code from the confirm phone SMS message. After success call, you can use the token in other mutations to prove that you have access to the phone number',
```

File: apps/condo/domains/user/schema/ConfirmPhoneActionService.js [hunk 4]
```diff
@@ -142,6 +150,13 @@ const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService
         {
             access: true,
             schema: 'startConfirmPhoneAction(data: StartConfirmPhoneActionInput!): StartConfirmPhoneActionOutput',
+            doc: {
+                summary: 'Creates ConfirmPhoneAction and sends confirmation message',
+                errors: pick(errors, [
```
Comment (pahaz) [inline apps/condo/domains/user/schema/ConfirmPhoneActionService.js:155]: we also need to add
```
{
  "errors": [
    {
      "message": "[security:tooManyRequests] resend timeout not expired",
      "name": "[security:tooManyRequests]",
      "path": [
        "startConfirmPhoneAction"
      ],
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "developerMessage": "[security:tooManyRequests] resend timeout not expired\n\nGraphQL request:2:3\n1 | mutation startConfirmPhone($fingerprint: String!, $phone: String!, $captcha: String!) {\n2 |   startConfirmPhoneAction(\n  |   ^\n3 |     data: {dv: 1, sender: {dv: 1, fingerprint: $fingerprint}, phone: $phone, captcha: $captcha}",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      },
      "data": {
        "time": "2022-05-26T12:20:43.735Z",
        "timeRemain": 34
      },
      "uid": "cl3mzhbo700eh0tkw2bz8bshd"
    }
  ],
  "data": {
    "startConfirmPhoneAction": null
  }
}
```

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/user/schema/ConfirmPhoneActionService.js (hunk 4 above)
Comment (AntonAL) [reply apps/condo/domains/user/schema/ConfirmPhoneActionService.js]: Added

**Review by pahaz — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/user/utils/serverSchema/guards.js [hunk 5]
```diff
@@ -33,8 +37,8 @@ class RedisGuard {
         const isLocked = await this.isLocked(lockName, action)
         if (isLocked) {
             const timeRemain = await this.lockTimeRemain(lockName, action)
-            throw new ApolloError(`${TOO_MANY_REQUESTS}]`, {
-                message: `${TOO_MANY_REQUESTS}] resend timeout not expired`,
+            throw new GQLError({
+                ...GQL_ERRORS.TOO_MANY_REQUESTS,
                 data: {
```
[the remark this answers is not in the document]
Comment (AntonAL) [reply apps/condo/domains/user/utils/serverSchema/guards.js]: Replaced

**Review by pahaz — APPROVED** [review]

**Review by akarjakin — APPROVED** [review]

**Pull request merged**